projects/fsastorefrontstyles/scss/overrides/header.scss

```diff
@@ -144,6 +144,9 @@ header {
           text-transform: uppercase;
         }
       }
+      .wrapper {
```

Comment:
I wouldn't consider this task as a bug at all. Spartacus has deliberately set lower z-index for this element. This is from their code comment:
```
      .wrapper {
        // we add a z-index since some of the elements on the page have (surprisingly) a z-index rule
        // we cannot make the z-index too high as it would conflict with the searchbox flyout.
        z-index: 20;
      }
```
Also it's not a good practice to have such a large z-index for anything other than perhaps dialog/popup boxes. I would say that in this case this is expected behavior and should not be changed.

Reply:
From a customer point of view, I would consider this as a bug, because this behavior blocks him to go on the desirable content.

We just have to think should I change z-index here or maybe the better solution is to change z-index of the dropdown on the Quote&Application page?